lib/openstack_query/docs/PROPERTIES.md

```diff
@@ -0,0 +1,117 @@
+| PROJECT_DOMAIN_ID   | `None`  | The ID of the domain which owns the project;                                                                                                                       |
+| PROJECT_ID          | `None`  | Unique ID Openstack has assigned the project.                                                                                                                      |
+| PROJECT_IS_DOMAIN   | `None`  | Indicates whether the project also acts as a domain. <br/>If set to True, the project acts as both a project and a domain.                                         |
+| PROJECT_IS_ENABLED  | `None`  | Indicates whether users can authorize against this project. <br/>if set to False, users cannot access project, additionally all authorized tokens are invalidated. |
+| PROJECT_NAME        | `None`  | Name of the project                                                                                                                                                |
+| PROJECT_PARENT_ID   | `None`  | The ID of the parent of the project.                                                                                                                               |
+
+#
+`FlavorProperties`
+
+
+| Property Enum      | Aliases | Description                                                                                     |
+|--------------------|---------|-------------------------------------------------------------------------------------------------|
+| FLAVOR_DESCRIPTION | `None`  | The description of the flavor                                                                   |
+| FLAVOR_DISK        | `None`  | Size of the disk this flavor offers. Type: int                                                  |
+| FLAVOR_EPHEMERAL   | `None`  | Size of the ephemeral data disk attached to this server. Type: int                              |
+| FLAVOR_ID          | `None`  | Unique ID Openstack has assigned the flavor.                                                    |
+| FLAVOR_IS_DISABLED | `None`  | Indicates whether flavor is disabled. <br/>True if disabled, False if not                       |
+| FLAVOR_IS_PUBLIC   | `None`  | Indicates if flavor is available to all projects. <br/>True if publicly available, False if not |
+| FLAVOR_NAME        | `None`  | Name of the flavor                                                                              |
+| FLAVOR_RAM         | `None`  | The amount of RAM (in MB) this flavor offers. Type: int                                         |
+| FLAVOR_SWAP        | `None`  | Size of the swap partitions.                                                                    |
+| FLAVOR_VCPU        | `None`  | The number of virtual CPUs this flavor offers. Type: int                                        |
+
+
+## Adding a property related to an existing resource
```

Comment:
This can stay in the original file if we split out the above sections though as it refers to the generic sense